### ISSUE TITLE
The course list seems not to be removed when user is logged out

### ISSUE THREAD

**Issue by phish108** [ISN-Zurich/ISN-Learning-Cards#69]
Current behaviour: 
- I log in as christian and get my course list correctly with 2 courses
- I log out
- I log in as isabella  and get my course list again with 2 courses instead of isabella's 3 courses.

In both cases the settings screen shows the correct information. 

Two things might go wrong. 
1. the client uses the local storage of the previous logged in user
2. the server gets wrong session information from the client that is assigned to the wrong user

Desired behaviour: 

get the course list for the current user.

**Comment by Bellina** [comment]
I was also registered only for 2 courses (for testing reasons I unsubscribed myself yesterday from one course)! 

Now I joined the third course again. On my phone I get all 3 courses now.

**Comment by ISN-Zurich** [comment]
Interestingly i got also three courses when logging in with my account.

Only when i logout/login a second time i get my courses.

Is the local storage completely cleared on logout?

Dr. Christian Glahn
Instructional and Mobile Design
ISN Zürich
 076 845 4205

On 15.09.2012, at 10:58, "Bellina" <notifications@github.com<mailto:notifications@github.com>> wrote:

I was also registered only for 2 courses (for testing reasons I unsubscribed myself yesterday from one course)!

Now I joined the third course again. On my phone I get all 3 courses now.
